pkg/helpers/array.go

```diff
@@ -19,6 +19,12 @@ func ConvertSlice[To, From any](slice []From, convert func(From) To) []To {
 	return ConvertToArray(convert, slice...)
 }
 
+func ReferenceSliceItems[T any](slice []T) []*T {
```

Comment:
Is that actually used anywhere? I can't find any references.
Also, I'm personally not a fan of adding functions which are essentially simple function compositions, although that's unfortunately already a practice in this package. 
https://github.com/scylladb/scylla-operator/pull/1388 moves the slices functions to a separate subpackage and so `slices.ConvertSlice(from, pointer.Ptr[T])` (or preferably `slices.Map(from, pointer.Ptr[T])`) seem neater and more self-explanatory to me

Reply:
seems like I've deleted the caller after all, dropped, thx